README.md

```diff
@@ -25,6 +25,7 @@
     - [Pure links](#pure-links)
     - [Wikilinks...](#wikilinks)
   - [Sub and Sup HTML Elements](#sub-and-sup-html-elements)
+  - [Mathematical expressions](#mathematical-expressions)
```

Comment:
I couldn't find the readme task, so I mirrored the changes from moduledoc manually. Let me know if/how I should generate it :)

Reply:
Done it is `mix xtra` I guess I should document that somewhere, maybe contribution notes/rules?

Reply:
Also done :blush:

Reply:
Ah it comes from an external package, that makes sense, thanks!